labs/hlf-operator.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: default
+title: Hyperledger Fabric Operator for Kubernetes
+parent: Labs
+---
+# Lab Name
+Hyperledger Fabric Operator for Kubernetes
+
+# Short Description
+Manage Hyperledger Fabric components at scale with a Kubernetes Operator.
+
+# Scope of Lab
+I think there's a gap to deploy a Hyperledger Fabric network in Kubernetes, this project automates to automate the deployment.
+
+The work will be focused on automating the creation of peers, orderers and CAs.
+
+I have some ideas I would like to share such as developing a User Interface, and I think this could be integrated natively with the [fabric-operations-console](https://github.com/hyperledger-labs/fabric-operations-console) which was recently released.
+
+# Initial Committers
+Github IDs for the set of initial committers.
+- https://github.com/dviejopomata
+- https://github.com/dviejokfs
+
+# Sponsor
+- https://github.com/dviejopomata
```

Comment:
[Note on the sponsor section] The sponsor requirement is now an optional. A well connected member in the community (either TSC, SIG/WG Chair, Maintainer of a project) could be sponsor for the new Labs project. The idea was to introduce new members to the rest of the community and guide them through.

Reply:
Added, though, It would be interesting to see if there's interest by HLF users, since probably the project needs a sponsor, do you know how to approach this?

Reply:
Sure, easiest way you can reach out to maintainers of a project is to find their mail ids in the `MAINTAINERS.md` file of the project repository. I am sure Labs stewards here will be happy to help. I am happy to guide you through as well. My RocketChat goes by the same name @arsulegai .
There was another proposal in the past which is similar to what I see here. Maybe this proposal has potential to grow community, integrate with other projects.